processor/filterprocessor/config.go

```diff
@@ -85,6 +91,58 @@ const (
 	Regexp = LogMatchType(filterset.Regexp)
 )
 
+var severityToNumber = map[string]plog.SeverityNumber{
```

Comment:
Since the data model defines severities as the numbers 0-24, we should support these as configuration values. I think you can just extend this map to include `"1": plog.SeverityNumberTRACE`, etc.

Reply:
Good idea; Added numeric severities.